src/app-layout/visual-refresh-toolbar/toolbar/trigger-button/styles.scss

```diff
@@ -106,6 +106,10 @@ handleSplitPanelMaxWidth function in the context.
   border-end-end-radius: 50%;
 }
 
+.trigger-wrapper-tooltip-visible {
+  /* used in test-utils*/
```

Comment:
To be precise, we are using these classes in our component tests but not in test utils (i.e, we are not providing test util wrappers for these elements)